### DISCUSSION TITLE
post/2023/2023-04-adding-citation-to-a-github-repository/

### DISCUSSION THREAD

**Discussion by giscus** [zyrikby/website_related#32] [Announcements]
# post/2023/2023-04-adding-citation-to-a-github-repository/

Following the principles of open science, it is typical for computer science researchers to share source code, tools or datasets accompanying a research paper. Being one of the most popular platforms, GitHub is often used to fulfill this task. While finding the repository associated with a paper usually is not difficult (researchers share the link in the camera-ready version of their paper), the reverse task - the discovery of the associated paper - is a more complicated task. To facilitate this process, researchers often add the citation to the paper to the main README file. However, for quite a while, GitHub facilitates this task allowing one to create a special citation file. The GitHub platform checks the presence of this file in each repository and adds a dropdown button with citation options if it finds it there. In this article, I explain how to add such citation files to your repository.

https://zhauniarovich.com/post/2023/2023-04-adding-citation-to-a-github-repository/

**Comment by VipinMadhaan** [comment]
will try

**Comment by KennethEnevoldsen** [comment]
Thanks for this guide!
